readalongs/align.py

```diff
@@ -509,30 +504,41 @@ def save_readalong(  # noqa C901
 
     output_base = os.path.join(output_dir, output_basename)
 
-    if text_grid:
+    # Create textgrid object if outputting to TextGrid or eaf
+    if "TextGrid" in output_formats or "eaf" in output_formats:
         audio = read_audio_from_file(audiofile)
         duration = audio.frame_count() / audio.frame_rate
         words, sentences = return_words_and_sentences(align_results)
         textgrid = write_to_text_grid(words, sentences, duration)
+
+    if "TextGrid" in output_formats:
         textgrid.to_file(output_base + ".TextGrid")
+
+    if "eaf" in output_formats:
         textgrid.to_eaf().to_file(output_base + ".eaf")
```

Comment:
It's logically identical, but I would find it easier to read if these two if statements were indented under the above if TextGrid or eaf block. It makes it more obvious we're still dealing with that type of output.

ditto the if srt and if vtt below, I'd put them inside the if srt or vtt block.

Reply:
sure, either way doesn't bug me.

Reply:
change done and pushed.